run.sh

```diff
@@ -154,7 +167,7 @@ fi
 
 echo "Launching the container..."
 CONTAINER_NAME=aiidalab
-CONTAINER_ID=`docker run -d -p "${_arg_port}:8888" -e JUPYTER_TOKEN="${_arg_token}" -v "${_arg_home_dir}:/home/aiida" --name ${CONTAINER_NAME} "${_arg_image}"`
+CONTAINER_ID=`docker run -d -p "${_arg_port}:8888" -e JUPYTER_TOKEN="${_arg_token}" -e PYTHONPATH="/home/${_arg_aiidalab_user}"  -e SYSTEM_USER="${_arg_aiidalab_user}" -e AIIDALAB_HOME="/home/${_arg_aiidalab_user}" -e AIIDALAB_APPS="/home/${_arg_aiidalab_user}/apps" -v "${_arg_home_dir}:/home/${_arg_aiidalab_user}" --name ${CONTAINER_NAME} "${_arg_image}"`
```

Comment:
Once https://github.com/aiidalab/aiidalab-home/pull/73 is merged and new version of aiidalab-home is released we should apply the following change:

```suggestion
CONTAINER_ID=`docker run -d -p "${_arg_port}:8888" -e JUPYTER_TOKEN="${_arg_token}" -e SYSTEM_USER="${_arg_aiidalab_user}" -e AIIDALAB_HOME="/home/${_arg_aiidalab_user}" -e AIIDALAB_APPS="/home/${_arg_aiidalab_user}/apps" -v "${_arg_home_dir}:/home/${_arg_aiidalab_user}" --name ${CONTAINER_NAME} "${_arg_image}"`
```